R/pkg/R/generics.R

```diff
@@ -439,6 +439,10 @@ setGeneric("covar_samp", function(col1, col2) {standardGeneric("covar_samp") })
 #' @export
 setGeneric("covar_pop", function(col1, col2) {standardGeneric("covar_pop") })
 
+#' @rdname dapply
+#' @export
+setGeneric("dapply", function(x, func, schema = NULL) { standardGeneric("dapply") })
```

Comment:
if schema is NULL, the result will be a single column (binary) DataFrame, what can we do on that?

Can we always ask for a schema, or infer the schema (may trigger a job)?